Tests/SwiftFormatRulesTests/OrderedImportsTests.swift

```diff
@@ -127,4 +127,38 @@ public class OrderedImportsTests: DiagnosingTestCase {
                 @testable import MyModuleUnderTest
                 """)
   }
+
+  public func testFileHeader() {
```

Comment:
Should we also have tests for the linting logic?

Reply:
I modified the first test function to include linter assertions. If these look good to you, I'll modify the remaining tests.

Reply:
Thanks! Go ahead and add the others, then merge it.